cosmos/converter.py

```diff
@@ -196,6 +203,31 @@ def validate_changed_config_paths(
         )
 
 
+def override_configuration(
```

Comment:
Do we need some separate tests for this function, or is it getting covered by some different tests?

Reply:
@pankajastro @pankajkoti could we add a follow up ticket so I can address this after? Due to illness I have not been able to improve the test coverage and we should not release 1.9 without this

Reply:
yes, created a follow-up ticket for this https://github.com/astronomer/astronomer-cosmos/issues/1540